app/views/reviews/_form.html.slim

```diff
@@ -161,30 +161,30 @@
       br
       .row.average-stat
         .col-xs-4
-          span.glyphicon.glyphicon-book
+          span=image_tag("icons/reading.png", size: "20")
           span
-            |  Reading:
+            |  &nbsp Reading:
```

Comment:
It's best to deal with this sort of spacing within styles, rather than hard-coding it in html. In this case, perhaps be could create a class for the icons and set `padding-right: 5px` or something.